### PR TITLE
[Jormungandr] fix: correction in the loading of environment variables

### PR DESCRIPTION
**Pull request by azime**
**Before correction:**

BSS_PROVIDER=[[{}, ....]]

**After Correction:**

BSS_PROVIDER=[{}, ....]

### CODE REVIEW

File: source/jormungandr/jormungandr/default_settings.py [hunk 1]
```diff
@@ -77,19 +77,11 @@
 
 # Bike self-service configuration
 # This should be moved in a central configuration system like ectd, consul, etc...
-BSS_PROVIDER = []
+BSS_PROVIDER = json.loads(os.getenv("JORMUNGANDR_BSS_PROVIDER_", "[]"))
```
Comment (xlqian) [inline source/jormungandr/jormungandr/default_settings.py:80]:
```suggestion
BSS_PROVIDER = json.loads(os.getenv("JORMUNGANDR_BSS_PROVIDER", "[]"))
```

File: source/jormungandr/jormungandr/default_settings.py [hunk 2]
```diff
@@ -77,19 +77,11 @@
...
 # Car parking places availability service
-CAR_PARK_PROVIDER = []
+CAR_PARK_PROVIDER = json.loads(os.getenv("JORMUNGANDR_CAR_PARK_PROVIDER_", "[]"))
```
Comment (xlqian) [inline source/jormungandr/jormungandr/default_settings.py:82]:
```suggestion
CAR_PARK_PROVIDER = json.loads(os.getenv("JORMUNGANDR_CAR_PARK_PROVIDER", "[]"))
```

File: source/jormungandr/jormungandr/default_settings.py [hunk 3]
```diff
@@ -77,19 +77,11 @@
...
-EQUIPMENT_DETAILS_PROVIDERS = []
-
-for key, value in os.environ.items():
-    if key.startswith('JORMUNGANDR_BSS_PROVIDER_'):
-        BSS_PROVIDER.append(json.loads(value))
-    elif key.startswith('JORMUNGANDR_CAR_PARK_PROVIDER_'):
-        CAR_PARK_PROVIDER.append(json.loads(value))
-    elif key.startswith('JORMUNGANDR_EQUIPMENT_DETAILS_PROVIDER_'):
-        EQUIPMENT_DETAILS_PROVIDERS.append(json.loads(value))
+EQUIPMENT_DETAILS_PROVIDERS = json.loads(os.getenv("JORMUNGANDR_EQUIPMENT_DETAILS_PROVIDER_", "[]"))
```
Comment (xlqian) [inline source/jormungandr/jormungandr/default_settings.py:84]:
```suggestion
EQUIPMENT_DETAILS_PROVIDERS = json.loads(os.getenv("JORMUNGANDR_EQUIPMENT_DETAILS_PROVIDER", "[]"))
```

File: source/jormungandr/jormungandr/default_settings.py (hunk 1 above)
Comment (azime) [reply source/jormungandr/jormungandr/default_settings.py:80]: I use the same [variable](https://github.com/CanalTP/navitia/pull/3699/files#diff-304d2bcf38a95031928c627b475cd0d63a98ac966e68649ad7f710457409287eL91)

**Review by xlqian — APPROVED** [review]

**Review by kadhikari — APPROVED** [review]

**Pull request merged**